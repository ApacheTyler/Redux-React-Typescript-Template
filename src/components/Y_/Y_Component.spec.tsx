import * as React from 'react'
import { shallow, mount, render } from 'enzyme'

import { createStore } from 'redux'

import Y from './Y_Component'


describe('<Y />', () => {

  it('renders', () => {
    expect(shallow(
      <Y welcomeMessage={'welcome to component y'} />
    )).toMatchSnapshot()
  });

})
