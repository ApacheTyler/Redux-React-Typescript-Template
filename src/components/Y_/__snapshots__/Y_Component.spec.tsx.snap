// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Y /> renders 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <div>
    <h1>
        welcome to component y
    </h1>
    <h2
        onClick={[Function]}
    />
</div>,
  "nodes": Array [
    <div>
      <h1>
            welcome to component y
      </h1>
      <h2
            onClick={[Function]}
      />
</div>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 0,
      "_context": Object {},
      "_currentElement": <Y
        welcomeMessage="welcome to component y"
/>,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": Y {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {
          "welcomeMessage": "welcome to component y",
        },
        "refs": Object {},
        "state": Object {
          "numberOfClicks": 0,
        },
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <div>
          <h1>
                    welcome to component y
          </h1>
          <h2
                    onClick={[Function]}
          />
</div>,
        "_debugID": 2,
        "_renderedOutput": <div>
          <h1>
                    welcome to component y
          </h1>
          <h2
                    onClick={[Function]}
          />
</div>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <Y
    welcomeMessage="welcome to component y"
/>,
}
`;
